src/tests/ll_verification.py

```diff
@@ -3781,6 +3781,72 @@ def ll_con_per_bv_81_c(transport, upperTester, lowerTester, trace):
 
     return success;
 
+# Implements LL/DDI/ADV/BI-05-C and LL/DDI/ADV/BI-06-C test cases (only difference is the event properties)
```

Comment:
I think these test cases should be placed next to the other LL/DDI/ADV test cases to not make this file messier than it already is.

Reply:
Moved